api/models/publication.js

```diff
@@ -31,7 +31,7 @@ const Publication = BaseModel.extend({
       id: attributes.id,
       url: attributes.url,
       title: attributes.title,
-      source: this.related('source').toJSON(),
+      source_id: this.attributes.source_id,
```

Comment:
I was expecting something like that, but I thought I'd double check with you before testing.

Why can't we just do `const attributes = this.attributes`? Might be a silly question, but I didn't get to run it yet.